modules/virt-configuring-disk-sharing-lun.adoc

```diff
@@ -26,9 +26,7 @@ For a LUN disk with an iSCSi connection and a persistent reservation, as require
 * The volume access mode must be `ReadWriteMany` (RWX) if the VMs that are sharing disks are running on different nodes.
 +
 If the VMs that are sharing disks are running on the same node, `ReadWriteOnce` (RWO) volume access mode is sufficient.
-
-* The storage provider must support a Container Storage Interface (CSI) driver that uses the SCSI protocol.
-
+* The storage provider must support a Container Storage Interface (CSI) driver that uses Fibre Channel (FC), Fibre Channel over Ethernet (FCoE), or iSCSI storage protocols.
```

Comment:
Is it worth spelling out iSCSI (and SCSI) in this module, too?

Reply:
Good question. In this case, though, I wouldn't spell it out, since it would make the sentence unnecessarily long.